app/helpers/commodities_helper.rb

```diff
@@ -216,17 +216,25 @@ def abbreviate_commodity_code(commodity)
   end
 
   def abbreviate_code(code)
-    case code.gsub(/0*\z/, '').length
+    only_code = code_without_subheading(code)
```

Comment:
Out of interest, is this used elsewhere?

Reply:
if you mean `only_code`?  it is used on line#221